tests/TestCase/Console/HelpFormatterTest.php

```diff
@@ -314,15 +314,15 @@ public function testXmlHelpWithChoices()
         $expected = <<<xml
 <?xml version="1.0"?>
 <shell>
-<name>mycommand</name>
-<description>Description text</description>
```

Comment:
Because there is no $parser->description('Description text') calls originally. Since the method is tested in the testXmlHelpDescriptionAndEpilog test, I don't think we need to double check here.